iocore/net/quic/QUICTLS_openssl.cc

```diff
@@ -373,7 +377,7 @@ QUICTLS::_process_post_handshake_messages(QUICHandshakeMsgs *out, const QUICHand
   size_t l = 0;
   ret      = SSL_read_ex(this->_ssl, data, 2048, &l);
 
-  if (ret < 0) {
```

Comment:
https://www.openssl.org/docs/man1.1.1/man3/SSL_read_ex.html
```
<= 0
The read operation was not successful, because either the connection was closed, an error occurred or action must be taken by the calling process. Call SSL_get_error(3) with the return value ret to find out the reason.
```